frontend/public/components/about-modal.tsx

```diff
@@ -41,7 +41,7 @@ const AboutModalItems: React.FC<AboutModalItemsProps> = ({ closeAboutModal, cv }
     group: ClusterVersionModel.apiGroup,
     resource: ClusterVersionModel.plural,
     verb: 'patch',
-    name: clusterVersion.metadata.name,
+    name: clusterVersion?.metadata?.name,
```

Comment:
Let's hard-code `'version`'. That way the access review is correct even if `clusterVersion` resource isn't defined. The resource name is an API, so it won't change.

```suggestion
    name: 'version',
```

Reply:
@spadgett maybe ?
```suggestion
    name: clusterVersion?.metadata?.name || 'version',
```